models/intermediate/int_zendesk__ticket_historical_satisfaction.sql

```diff
@@ -42,10 +73,34 @@ with satisfaction_updates as (
     left join latest_comment
         using(ticket_id)
 
-    left join latest_score
+    left join first_and_latest_score
```

Comment:
can you use "using" here, given that multiple tables have ticket_id? 

I think this works fine in BQ, but not the other warehouses (though I may be wrong)

Reply:
This does work fine in BQ and looks to run fine in RS and SF, but with our limited testing dataset I am not 100%. Removed "using" in joins just to be safe!